contracts/BaseStrategy.sol

```diff
@@ -152,6 +164,22 @@ abstract contract BaseStrategy {
 
     bool public emergencyExit;
 
+    // modifiers
+    modifier onlyAuthorized() {
+        require(msg.sender == strategist || msg.sender == governance(), "!authorized");
```

Comment:
This will be one of only 2 places that `governance()` is used, so try combining that call into here

Reply:
do you mean we want to replace the governance method for `vault.governance()` in the modifiers? and remove the method altogether?

Reply:
Yes!

Reply:
so i added the modifiers here 83f37c8, but still governance() is used in 4 places which i thought then its useful to keep, and also is handy for viewing governance address in view method and APIs